Assets/Scripts/Config/InterceptorOriginObject.cs

```diff
@@ -0,0 +1,153 @@
+using UnityEngine;
+
+// Runtime representation of an interceptor origin in the simulation.
+// This component manages the actual GameObject state and provides access to current
+// position/velocity. The associated InterceptorOriginConfig provides the static configuration data.
+public class InterceptorOriginObject : MonoBehaviour {
```

Comment:
suggestion: call it InterceptorOrigin since it's already an "object"

Reply:
resolved